package.json

```diff
@@ -1,44 +1,59 @@
 {
-  "name": "learn-starter",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "ins": "yarn && yarn dev"
-  },
-  "dependencies": {
-    "@chakra-ui/icons": "^1.0.13",
-    "@chakra-ui/react": "^1.6.4",
-    "@emotion/react": "^11.4.0",
-    "@emotion/styled": "^11.3.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-brands-svg-icons": "^5.15.4",
-    "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/react-fontawesome": "^0.1.15",
-    "@headlessui/react": "^1.2.0",
-    "@heroicons/react": "^1.0.1",
-    "axios": "^0.21.4",
-    "classnames": "^2.3.1",
-    "date-fns": "^2.22.1",
-    "framer-motion": "^4.1.17",
-    "gray-matter": "^4.0.3",
-    "next": "^12.0.8",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "rehype-highlight": "^5.0.0",
-    "rehype-raw": "^6.1.0",
-    "rehype-sanitize": "^5.0.0",
-    "rehype-stringify": "^9.0.2",
-    "remark-parse": "^10.0.0",
-    "remark-rehype": "^10.0.0",
-    "sharp": "^0.29.3",
-    "unified": "^10.1.0"
-  },
-  "devDependencies": {
-    "autoprefixer": "^10.2.6",
-    "postcss": "^8.3.5",
-    "tailwindcss": "^2.2.2"
-  }
```

Comment:
Why the heck is this? 
It seems normal on my VS Code
Perhaps, there's something wrong with prettier config